engine.py

```diff
@@ -222,31 +256,15 @@ def __getattr__(self, name):
                 if not hasattr(PySide, "__version__"):
                     PySide.__version__ = "<unknown>"
 
-                # tell QT to interpret C strings as utf-8
-                utf8 = QtCore.QTextCodec.codecForName("utf-8")
-                QtCore.QTextCodec.setCodecForCStrings(utf8)
-
-                # a simple dialog proxy that pushes the window forward
-                class ProxyDialogPySide(QtGui.QDialog):
-                    def show(self):
-                        QtGui.QDialog.show(self)
-                        self.activateWindow()
-                        self.raise_()
-
-                    def exec_(self):
-                        self.activateWindow()
-                        self.raise_()
-                        # the trick of activating + raising does not seem to be enough for
-                        # modal dialogs. So force put them on top as well.
-                        self.setWindowFlags(QtCore.Qt.WindowStaysOnTopHint | self.windowFlags())
-                        return QtGui.QDialog.exec_(self)
-                        
-                
                 base["qt_core"] = QtCore
                 base["qt_gui"] = QtGui
-                base["dialog_base"] = ProxyDialogPySide
-                self.log_debug("Successfully initialized PySide '%s' located in %s." 
+                self.log_debug("Successfully initialized PySide '%s' located in %s."
                                % (PySide.__version__, PySide.__file__))
+
+                # tell QT4 to interpret C strings as utf-8
+                utf8 = QtCore.QTextCodec.codecForName("utf-8")
```

Comment:
Just checking: we're not doing this for PySide2, is it needed? If not, could you add a comment in the the PySide2 case so we know that this is not needed?